required/latex-lab/testfiles-block/blocks-item-02.luatex.tlg

```diff
@@ -75,13 +75,35 @@ l. ...  \begin{center} X
 Try typing  <return>  to proceed.
 If that doesn't work, type  X <return>  to quit.
 ==> blockenv common ending on input line ...
+! LaTeX Error: Something's wrong--perhaps a missing \item.
```

Comment:
Is this new no-item error expected?

It seems now
https://github.com/latex3/latex2e/blob/70890a831c8cdef4da25002e1de543312809ba54/required/latex-lab/testfiles-block/blocks-item-02.lvt#L29-L35
raises three no-item errors.

Reply:
yes, well, let's say I accepted that as a side-effect for now. It is an error situation and it doesn't work that well in combination with tagging. I guess as some point one could revisit the mechanism and see how to make it better without slowing down the normal processing, but for now I think it is good enough.